pkg/tablecodec/tablecodec.go

```diff
@@ -977,7 +980,11 @@ func decodeHandleInIndexKey(keySuffix []byte) (kv.Handle, error) {
 	return kv.NewCommonHandle(keySuffix)
 }
 
-func decodeHandleInIndexValue(value []byte) (handle kv.Handle, err error) {
+// DecodeHandleInIndexValue decodes handle in unqiue index value.
+func DecodeHandleInIndexValue(value []byte) (handle kv.Handle, err error) {
+	if len(value) <= MaxOldEncodeValueLen {
+		return decodeIntHandleInIndexValue(value), nil
+	}
```

Comment:
Could we remove these codes that look like they will be processed on line988?

Reply:
This speeds up the function for simplest int handles(like `_tidb_rowid`).

Reply:
The performance difference is `9ns/op` compare `15ns/op`

Reply:
Got it.